ui/app/components/search-select.js

```diff
@@ -81,6 +82,10 @@ export default Component.extend({
     onChange(val) {
       this.onChange(val);
     },
+    createOption(optionId) {
+      let newOption = { name: optionId, id: optionId };
+      this.selectedOptions.pushObject(newOption);
```

Comment:
I think this needs to call `this.handleChange()` as well - anything added this way doesn't get saved on form save.